tests/baselines/reference/narrowingUnionToUnion.types

```diff
@@ -557,8 +557,8 @@ if (isEmpty(test)) {
 >     : ^^^^^^^^^^^^^^^^^^^^^^^^^
 
     test;  // EmptyString
->test : EmptyString

```

Comment:
Those changes are fine - it's the `candidate` that had `.origin` on it but the goal is to preserve `type`'s `.origin` and not `.candidate`'s. The contained types here are exactly the same